internal/cue/validate.go

```diff
@@ -71,7 +125,14 @@ func ValidateFiles(dst io.Writer, files []string) error {
 		// Quit execution of the cue validating against the yaml
 		// files upon failure to read file.
 		if err != nil {
-			return err
+			var sb strings.Builder
+
```

Comment:
do we need a string builder in this case?  String builder seems unnecessary when we arent programmatically adding to it.

Could we just do

```
fmt.Printf("X Validation failure!\n\n)
fmt.Printf("Failed to read file %q\n", f)
```

Reply:
@markphelps Yeah definitely unnecessary for a known amount of things that you want to write out. Only makes sense in the cases where we are writing out the error details.